openvpn-install.sh

```diff
@@ -133,6 +133,7 @@ if [[ -e /etc/openvpn/server.conf ]]; then
 			rm -rf pki/issued/$CLIENT.crt
 			rm -rf /etc/openvpn/crl.pem
 			cp /etc/openvpn/easy-rsa/pki/crl.pem /etc/openvpn/crl.pem
+			chmod nobody:nobody /etc/openvpn/crl.pem
```

Comment:
It would be better to use instead `chmod nobody:$NOGROUP /etc/openvpn/crl.pem` but 
```
# Find out if the machine uses nogroup or nobody for the permissionless group
if grep -qs "^nogroup:" /etc/group; then
        NOGROUP=nogroup
else
       	NOGROUP=nobody
fi
```
from [here](https://github.com/Angristan/OpenVPN-install/blob/89925cbbe854741d75f370be0525e8cf64b6c961/openvpn-install.sh#L383) should be moved on the top of the script